fastlane/Fastfile

```diff
@@ -666,6 +691,81 @@ platform :ios do
     Snapshot::ReportsGenerator.new.generate
   end
 
+  desc 'Downloads localized strings and App Store Connect metadata from GlotPress'
+  lane :download_localized_strings_and_metadata_from_glotpress do
+    download_localized_strings_from_glotpress
+    download_localized_metadata_from_glotpress
+  end
+
+  desc 'Downloads localized `.strings` from GlotPress'
+  lane :download_localized_strings_from_glotpress do
+    # FIXME: This is a copy of what the release toolkit `ios_update_metadata` action does.
+    # We'll soon replace this with the new `ios_download_strings_files_from_glotpress`-based workflow.
+    sh("cd #{PROJECT_ROOT_FOLDER} && ./Scripts/update-translations.rb")
+
+    files_to_commit = Dir.glob(File.join(RESOURCES_FOLDER, '**', '*.strings'))
+    git_add(path: files_to_commit, shell_escape: false)
+    git_commit(
+      path: files_to_commit,
+      message: 'Update translations',
+      allow_nothing_to_commit: true
+    )
+  end
+
+  desc 'Downloads localized metadata for App Store Connect from GlotPress'
+  lane :download_localized_metadata_from_glotpress do
+    metadata_directory = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'metadata')
+
+    # FIXME: We should make the `fastlane/metadata/default/release_notes.txt` path be the source of truth for the original copies in the future.
+    # (will require changes in the `update_appstore_strings` lane, the Release Scenario, the MC tool to generate the announcement post…)
+    #
+    # In the meantime, just copy the file to the right place for `deliver` to find, i.e. to the `default` pseudo-locale which is used as fallback
+    release_notes_source = File.join(RESOURCES_FOLDER, 'release_notes.txt')
+    FileUtils.cp(release_notes_source, File.join(metadata_directory, 'default', 'release_notes.txt'))
+
+    # FIXME: Replace this with a call to the future replacement of `gp_downloadmetadata` once it's implemented in the release-toolkit (see paaHJt-31O-p2).
+    locales_map = GLOTPRESS_TO_ASC_METADATA_LOCALE_CODES
```

Comment:
@AliSoftware I decided not to apply your suggestion in this instance because `locales_map` is used in two places in the action. Given that "reuse", it felt neater to keep the assignment even though it's technically unnecessary.

However, part of me is pushing to DRY and remove `locales_map`, so if you still feel that'd be better after reading this comment, I'll happily followup.

Reply:
Ah right I missed the second usage down the line with the gitkeep checks. Fine by me to keep as is then 👍